yaml/wp_energy_combined.yaml

```diff
@@ -40,7 +40,7 @@ esphome:
             id(${property_wh}).publish_state(value);
           });
           CallbackHandler::instance().addCallback(std::make_pair(${target},Property::k${property_kwh}),[](const SimpleVariant& value){
-            id(${sensor_name}).publish_state(static_cast<std::uint16_t>(value) + id(${property_wh}).state - id(gOFFSET_EL_AUFNAHMELEISTUNG_HEIZ_TAG_WH));
```

Comment:
Yep, that was obviously wrong :sweat_smile: There are more issues ... I will push a fix morrow. Need to see first if it works correctly :+1: